app/src/main/java/com/imashnake/animite/features/media/MediaPage.kt

```diff
@@ -127,7 +127,7 @@ fun MediaPage(
                         .fillMaxSize()
                 )
 
-                if (media.stats != null) {
+                if (!media.stats.isNullOrEmpty()) {
```

Comment:
Thanks I hate API returning empty or null randomly
cc @Iannnr fixy for us

Reply:
So true!

Reply:
[Related convo in AniList's Discord](https://discord.com/channels/210521487378087947/281216402684116993/1048030452398690394).